.env.development

```diff
@@ -1,2 +1,2 @@
-GATSBY_USERBASE_TEST_APP_ID=8fce5df2-a997-4ceb-9023-d277cc31d2d1
+GATSBY_USERBASE_TEST_APP_ID=8af6f24a-fcf3-4fa5-9e20-c8fe279c5b5c
```

Comment:
You should not change this, I have a developer account for us. If you need to change this you add a file named `.env.development.local` that you do not check-in where you can override stuff.

Reply:
OK
I added a file named` .env.development.local `now